src/api/java/com/minecolonies/api/util/InventoryUtils.java

```diff
@@ -133,7 +133,7 @@ public static Item getItemFromBlock(final Block block)
     @NotNull
     public static Boolean isItemStackEmpty(@Nullable final ItemStack stack)
     {
-        return stack == null || stack == EMPTY || stack.getCount() <= 0;
+        return stack == null || stack == EMPTY || stack.getItem() != null || stack.getCount() <= 0;
```

Comment:
should be getItem() == null, and, we should have that at least for direct compatibility to 1.10 (where this might happen)